### PR TITLE
profile VishalSundarrajan.md

### PR DESCRIPTION
**Pull request by VishalSundarrajan**
https://rawgit.com/VishalSundarrajan/VishalSundarrajan.github.io/vishal-profile/#!pages/vi/profiles/VishalSundarrajan.md

### CODE REVIEW

File: pages/vi/profiles/VishalSundarrajan.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
++ **Database** : SQL, Mongodb, NoSQL
++ **Cloud** : AWS (EC2, MariaDB), Git, Docker, Google Firebase
++ **Tools/Frameworks** : Django, Spring boot, Spring, Hibernate, Servlet, Catch2, googletest, Maven, pip
+
+
+## What are my hobbies?
+
+1. If I have enough time and food, I play chess all day.     
+
+    -Check out my elo [here](https://www.chess.com/member/vishalicious) and send me a game request if you think you are good. :fearful:
```
Comment (lmmrssa) [inline pages/vi/profiles/VishalSundarrajan.md]: please add space after `-`

Comment (VishalSundarrajan) [reply pages/vi/profiles/VishalSundarrajan.md]: Hey @lmmrssa. Fixed that. Also, I could not get emoji's work in the HTML page. Do I have to remove them?

**Comment by VishalSundarrajan** [comment]
I removed the last one but for some reason its not updated in rawgit (above link)

**Comment by VishalSundarrajan** [comment]
Its good now but quite strange why its not immediate. Should I consider posting this as an issue? It happened with my other branches in my fork. I also want to try adding them back after going through [this](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/1991 )

**Pull request merged**